packages/server/src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 ########################################################################################################################
 # Database configuration
 ########################################################################################################################
-spring.datasource.url=jdbc:postgresql://postgres.staging.terarium.ai:5432/terarium
+spring.datasource.url=jdbc:postgresql://10.64.22.49:5432/terarium
```

Comment:
Why does PostGres do not have its dev url?

Reply:
doesn't need it. ES and keycloak needed it because of SSL certs, but MQ and PG do not

Reply:
That makes sense, even if it disturb my need to organise everything neatly.